{"name": "test_failed_login", "status": "passed", "description": "../features/login.feature: Failed login with invalid credentials", "start": 1759216294585, "stop": 1759216294855, "uuid": "56a6d6c0-6604-400a-a02b-00148f0bdfc2", "historyId": "eb98303b78b0dd8a81a7d4c9688110a7", "testCaseId": "eb98303b78b0dd8a81a7d4c9688110a7", "fullName": "tests.test_login#test_failed_login", "labels": [{"name": "story", "value": "2.Failed login"}, {"name": "feature", "value": "Login flow"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "MacBookPro"}, {"name": "thread", "value": "4145-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}], "titlePath": ["tests", "test_login.py"]}